torchreid/integration/sc/parameters.py

```diff
@@ -56,6 +56,32 @@ class __LearningParameters(ParameterGroup):
             affects_outcome_of=ModelLifecycle.TRAINING
         )
 
+    @attrs
+    class __InferenceParameters(ParameterGroup):
+        header = string_attribute("Parameters for inference")
+        description = header
+
+        class_name = selectable(default_value=Models.CLASSIFICATION,
+                                header="Model class for inference",
+                                description="Model classes with defined pre- and postprocessing",
+                                editable=False,
+                                visible_in_ui=True)
```

Comment:
`class_name` looks like an internal implementation parameter, which is not configurable. Why do we need it here?

Reply:
`class_name` it is type of using model wrapper. For any new model with own pre- and postprocessing you create `template.yml` file in which you should override this parameter. So I add `class_name ` to `configuration.yaml` and `parameters.py`